guides/v2.1/comp-mgr/patching.md

```diff
@@ -0,0 +1,63 @@
+---
+group: software-update-guide
+title: Applying patches
+functional_areas:
+  - Upgrade
+---
+
+This topic discusses the workflow for applying a patch to your Magento application.
+
+We strongly recommend that patches be tested in a staging/development environment before being deployed to production.
+
+## First: backup your data
+
+We also strongly recommend that you backup your data before any patches are applied.
+To back up your data, please refer to [Back up and roll back the file system][].
+
+## Get the patch
+
+Magento patches can be found in the [Magento Security Center][]. Follow the instructions on the page to download the patch file, depending on your version and installation type.
+
+## How patches work
+
+Patch (or diff) files are text files that note:
+
+* The file(s) to be changed.
```

Comment:
Probably don't need periods for the items on this list since they are not complete sentences

Reply:
I debated that...